CHANGELOG.md

```diff
@@ -8,10 +8,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [v1.10.2]
```

Comment:
Just leave this line out for now and keep it as unreleased for testing?

Reply:
Would also be cool to keep the usual changelog thingy of `Added` / `Changed` / `Fixed`

Example https://github.com/giantswarm/aws-operator/blob/master/CHANGELOG.md

Reply:
I updated the changelog and had fluentd start locally. I am not sure I can test it as the CI is only set-up on the master branch. Should I add a docker build/push latest on all the branches so I can test it on a cluster?

Reply:
My bad, it is build on every commit :(